clinica/pipelines/statistics_volume_correction/statistics_volume_correction_utils.py

```diff
@@ -155,16 +191,17 @@ def produce_figures(nii_file, template, type_of_correction, t_thresh, c_thresh,
     ]
 
 
-def generate_output(t_map, figs, name):
-    """
-        Produce output
-    Args:
-        t_map: (str) path to t-map on which whole pipeline was based
-        figs: (list of str) paths to figs to save
-        name: (str) name of the correction (ex: cluster_correction_FWE)
-
-    Returns:
-        Nothing
+def generate_output(t_map: str, figs: list, correction_name: str) -> None:
+    """Produce output
```

Comment:
I know it was like this before, but the docstring is even less informative than the function's name... 😱 
Maybe explain what is produced ?